sdk/src/client/api/block_builder/mod.rs

```diff
@@ -43,12 +44,18 @@ impl ClientInner {
                 issuance.latest_finalized_slot,
                 issuer_id,
             ),
-            // TODO: burned mana calculation
-            BlockBody::build_basic(issuance.strong_parents()?, 0)
-                .with_weak_parents(issuance.weak_parents()?)
-                .with_shallow_like_parents(issuance.shallow_like_parents()?)
-                .with_payload(payload)
-                .finish_block_body()?,
+            BlockBody::build_basic(
+                issuance.strong_parents()?,
+                MaxBurnedManaAmount::MinimumAmount {
+                    params: protocol_params.work_score_parameters,
+                    reference_mana_cost: issuance.commitment.reference_mana_cost(),
+                },
+            )
+            .with_weak_parents(issuance.weak_parents()?)
+            .with_shallow_like_parents(issuance.shallow_like_parents()?)
+            .with_payload(payload)
+            .finish()?
```

Comment:
Can you use `finish_block_body` instead of finish + into pls

Reply:
Done